src/ResourceManager/Compute/Commands.Compute/Generated/Disk/DiskCreateOrUpdateMethod.cs

```diff
@@ -62,7 +62,7 @@ protected object CreateDiskCreateOrUpdateDynamicParameters()
 
             var pDisk = new RuntimeDefinedParameter();
             pDisk.Name = "Disk";
-            pDisk.ParameterType = typeof(PSDisk);
```

Comment:
@hyonholee do the changes made to this file revert the breaking change mitigation from last release? (_i.e._, using `PSDisk` instead of `Disk`)